idunn/datasources/directions/hove/models.py

```diff
@@ -217,7 +217,11 @@ def as_api_route_summary_part(self) -> api.RouteSummaryPart:
                 if self.display_informations
                 else None
             ),
-            distance=42,
```

Comment:
Wow, how did I miss that !

Reply:
I though about checking all TODOs but didn't check for the remaining 42's :see_no_evil: